### ISSUE TITLE
Create an endpoint to get a price of a room

### ISSUE THREAD

**Issue by beogip** [windingtree/crypto-booking#358]
We need to have an endpoint that returns the price of a room using X token.
ie:
```
/prices/:roomType?token=LIF
```

**Comment by PabloAlice** [comment]
wouldn't be better to have the chance of asking for multiple quotations at the same request?
```
/prices/:roomType?quotations=[LIF]
```

**Comment by JirkaChadima** [comment]
The API right now does not support multiple currencies and does not support conversions from fiat. And it's probably not coming soon.

So the booking app can consume the price in fiat and do the conversion client side or we have to wait with this feature for the backend implementation. We have proposed this as [one of the challenges](https://github.com/ethberlin-hackathon/ETHBerlin-Teambuilding/issues/66) for ETH Berlin but it seems that nobody took it.

**Comment by JirkaChadima** [comment]
I have just realized that we can store the price in multiple currencies in our data model, the only problem we have is the currency conversion. Every ratePlan can contain a currency field and we're limiting that only on the data specification level and validatoin. So in theory, it can contain even crypto symbols.

**Comment by beogip** [comment]
@JirkaChadima, currently in crypto-booking the price is pulled from `wt-read-api` from FE and BE. The propose of this issue is to only consume `wt-read-api` from BE and expose an endpoint with that price. Makes sense?
Sometimes we'll want to apply discounts and that logic must be in one place

**Comment by JirkaChadima** [comment]
Yeah, that makes perfect sense. I agree that this should be in one place.
